canvasapi/assignment.py

```diff
@@ -254,6 +329,27 @@ def set_extensions(self, assignment_extensions, **kwargs):
             for extension in extension_list
         ]
 
+    def show_provisonal_grades_for_student(self, anonymous_id, **kwargs):
+        """
+        :call: `GET /api/v1/courses/:course_id/assignments/:assignment_id/
+            anonymous_provisional_grades/status \
+        <https://canvas.instructure.com/doc/api/all_resources.html#method.anonymous_provisional_grades.status>`_
+        :param anonymous_id: The id of the student to show the status for
+        :type anonymous_id: string
+        :rtype: dict
+        """
+        request = self._requester.request(
+            "GET",
+            "courses/{}/assignments/{}/anonymous_provisional_grades/status/{}".format(
```

Comment:
Same as `get_provisional_grades_status` above, there is no info in the url after `/status`. `anonymous_id` must be passed as a parameter.